lfsapi/endpoint_finder.go

```diff
@@ -151,8 +151,8 @@ func (e *endpointGitFinder) NewEndpointFromCloneURL(operation, rawurl string) lf
 		return ep
 	}
 
-	if strings.HasSuffix(rawurl, "/") {
-		ep.Url = rawurl[0 : len(rawurl)-1]
+	if strings.HasSuffix(ep.Url, "/") {
```

Comment:
Thanks for the quick fix! `strings.TrimSuffix` would be appropriate here too.

Reply:
That is a good idea, but I'm going to leave this as it stands for now in the interests of making the minimum possible fix.